frontend/packages/integration-tests-cypress/tests/app/admission-webhook-warning-notifications.cy.ts

```diff
@@ -0,0 +1,172 @@
+metadata:
+  name: ${DEPLOY_NAME}
+  annotations: {}
+  namespace: ${testName}
+spec:
+  selector:
+    matchLabels:
+      app: deploy1
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: deploy1
+    spec:
+      containers:
+        - name: ${CONTAINER_NAME}
+          image: >-
+            image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          env:
+            - name: app
+              value: frontennd
+      imagePullSecrets: []
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  paused: false
+`;
+
+describe('Admission Webhook warning notification', () => {
+  before(() => {
+    cy.login();
+    cy.createProjectWithCLI(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.visit('/');
+    cy.deleteProjectWithCLI(testName);
+  });
+
+  it('Create a pod and display Admission Webhook warning notification', () => {
+    cy.visit(`/k8s/ns/${testName}/import`);
+    yamlEditor.isImportLoaded();
+    yamlEditor.setEditorContent(pod1ReqObj).then(() => {
+      cy.intercept('POST', `/api/kubernetes/api/v1/namespaces/${testName}/pods`, (req) => {
+        req.continue((res) => {
+          res.headers = {
+            Warning: WARNING_FOO,
+          };
+        });
+      }).as(POD_CREATED_ALIAS);
+      yamlEditor.clickSaveCreateButton();
+      cy.wait(`@${POD_CREATED_ALIAS}`, WAIT_OPTION);
+      detailsPage.sectionHeaderShouldExist('Pod details');
+      cy.byTestID(LEARN_MORE_ID).parents().contains('Admission Webhook Warning');
+      cy.byTestID(LEARN_MORE_ID)
+        .parents()
+        .contains(`Pod ${POD_NAME}-a violates policy ${WARNING_FOO}`);
+      cy.byTestID(LEARN_MORE_ID).contains('Learn more').click();
+    });
+  });
+
+  it('Create bulk resources and display Admission Webhook warning notifications', () => {
+    cy.visit(`/k8s/ns/${testName}/import`);
+    yamlEditor.isImportLoaded();
+    yamlEditor.setEditorContent(bulkResourcesReqObj).then(() => {
+      for (const resource of resources) {
+        cy.intercept(
+          'POST',
+          `/api/kubernetes/${resource.path}/v1/namespaces/${testName}/${resource.resource}`,
+          (req) => {
+            req.continue((res) => {
+              res.headers = {
+                Warning: resource.warning,
+              };
+            });
+          },
+        ).as(BULK_RESOURCES_CREATED_ALIAS);
+      }
+      yamlEditor.clickSaveCreateButton();
+      cy.wait(`@${BULK_RESOURCES_CREATED_ALIAS}`, WAIT_OPTION);
+      cy.byTestID('resources-successfully-created').contains('Resources successfully created');
+      cy.byTestID(LEARN_MORE_ID).parents().contains('Admission Webhook Warning');
+      cy.byTestID(LEARN_MORE_ID)
+        .parents()
+        .contains(`Pod ${POD_NAME}-b violates policy ${WARNING_FOO}`);
+      cy.byTestID(LEARN_MORE_ID)
```

Comment:
Use [data-test](https://github.com/openshift/console/blob/a4e560ac5d50c94fc36b351d109bc1db63968121/frontend/packages/console-shared/src/components/toast/ToastProvider.tsx#L61) on the `<Alert>` itself.